src/main/java/edu/cornell/kfs/concur/service/impl/ConcurReportsServiceImpl.java

```diff
@@ -31,22 +33,60 @@
 import edu.cornell.kfs.concur.rest.xmlObjects.ExpenseEntryDTO;
 import edu.cornell.kfs.concur.rest.xmlObjects.ExpenseReportDetailsDTO;
 import edu.cornell.kfs.concur.rest.xmlObjects.ItemizationEntryDTO;
-import edu.cornell.kfs.concur.service.ConcurAccessTokenService;
 import edu.cornell.kfs.concur.service.ConcurReportsService;
 import edu.cornell.kfs.sys.CUKFSConstants;
 import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
 import edu.cornell.kfs.sys.service.impl.DisposableClientServiceImplBase;
 import edu.cornell.kfs.sys.util.CURestClientUtils;
 
 public class ConcurReportsServiceImpl extends DisposableClientServiceImplBase implements ConcurReportsService, DisposableBean {
-    
-	private static final Logger LOG = LogManager.getLogger(ConcurReportsServiceImpl.class);
-    protected ConcurAccessTokenService concurAccessTokenService;
+
+    private static final Logger LOG = LogManager.getLogger(ConcurReportsServiceImpl.class);
+
+    private static final String BEARER_AUTHENTICATION_SCHEME = "Bearer";
```

Comment:
we have a constant for this: ConcurConstants.BEARER_AUTHENTICATION_SCHEME

Reply:
The constant ConcurConstants.BEARER_AUTHENTICATION_SCHEME was added by the KFSPTS-22326 changes, which are not in PROD yet and will be going out with the 08/28/2022 patch instead. Since I intend to use this PR's commit for the emergency patch branch, I would prefer to use this separate constant to avoid merge conflicts. We could clean this up at a future date if needed.

Please let me know if you have a suggestion on an alternate way to handle this.

Reply:
that makes total sense, I agree with sticking with this local constant.  Thanks for explaining